site/survey.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="kaos620">
+  <meta name="description" content="This page comes with the objective of completing the second assigment of CSD112">
+  <meta>
+  <title>My GitHub Page</title>
+  <link rel="stylesheet" href="styles/main.css">
+</head>
+<body>
+  <div class="width-wrap">
+    <header id="top">
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="links.html">Links</a></li>
+          <li><a href="glossary.html">Glossary</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+        <div id="survey">
+            <h1>Survey</h1>
+            <form>
+                
+                <p>
+                <label for="name_field">What is your name? <span aria-label="required">*</span></label>
+                <input type="text" id="name_field" name="name_field" required>
+                </p>
+                
+                <p>
+                <label for="email_field">What is your email? <span aria-label="required">*</span></label>
+                <input type="email" id="email_field" name="email_field" required>
+                </p>
+                
+                <p>
+                <label for="best_whole_number">Which whole number is best?</label>
+                <input type="number" id="best_whole_number" name="best_whole_number" min="0" step="1">
+                </p>
+                
+                <p>
+                <label for="best_day">Which day is best?</label>
+                <input type="date" id="best_day" name="best_day">
+                </p>
+                
+                <p>
+                <label for="best_bear">Which bear is best? <span aria-label="required">*</span></label>
+                <select id="best_bear" name="best_bear" required>
+                    <option value="">&nbsp;</option>
```

Comment:
9. `Lab 4 > Forms > has <form> element > with the following controls > A dropdown list having > options > empty (default)`
```suggestion
                    <option value="" selected>&nbsp;</option>
```